### ISSUE TITLE
EnderChest items are deletet

### ISSUE THREAD

**Issue by note3crafter** [thebigsmileXD/BedWars#11]
When i join a BedWars Round deletet this the EnderChestItems from the Players this is not good this is bad :/ can you fix it?

**Comment by thebigsmileXD** [comment]
I explicitly clear all inventories so the player does not cheat items into bw. I could send a fake clear inventory which should help with that. Will think about it

**Comment by note3crafter** [comment]
No, how should one of the one enenderchest cheats? As long as it is not installed in the map you can also get nothing out of the box ... Could dud as den in so that you can make it adjustable in the confiq? So that it will be clever or not I think better

**Comment by note3crafter** [comment]
And if you fall out of the map you die and fall out of the world although the bed is still there the game unfortunately will finish but sunniest this plugin has succeeded very well

**Comment by thebigsmileXD** [comment]
Make a new issue for that please

Will introduce Team Enderchests soon
